src/pepr/operator/controllers/network/generate.ts

```diff
@@ -52,6 +53,8 @@ function getPeers(policy: Allow): V1NetworkPolicyPeer[] {
     }
 
     peers.push(peer);
+  } else if (policy.remoteCidr !== undefined) {
+    peers = [remoteCidr(policy.remoteCidr)];
```

Comment:
Could we toss a remoteCidr policy in one of our test packages? Doesn't need to be "tested" beyond the package being ready, just would like to make sure we have one somewhere in our code.

Reply:
Added generate.spec.ts tests as well as a new netpol for the test-tenant-app that utilizes the remoteCidr.